include/bluetooth/nrf/host_extensions.h

```diff
@@ -61,7 +61,7 @@ struct bt_conn_set_pcr_params {
 	 */
 	uint16_t beta;
 	/** The lower limit of the RSSI golden range. The RSSI golden range is explained in
-	 *  Core_v5.4, Vol 6, Part B, Section 5.1.17.1.
+	 *  Bluetooth Core Specification Vol 6, Part B, Section 5.1.17.1.
```

Comment:
```suggestion
	 *  `Bluetooth Core Specification`_ Vol 6, Part B, Section 5.1.17.1.
```
That makes clickable link